### ISSUE TITLE
SQL parse error when a semi-colon ";" is used in a property name.

### ISSUE THREAD

**Issue by Cyberider** [orientechnologies/orientdb#5641]
When a semi-colon ```;``` is used in a class property name, some functionality (SQL statements using that property name) are lost, throw errors.

This bug is easy to reproduce. **Warning: the only way to recover from this bug, back to normal, is to drop the class. You must use the web Studio, Browse tab, to enter your SQL statements.**

In any database, such as the demo greatfuldeadconcerts, create a new property with any name, rename that property to a new name with a ```;``` at the end of that new name. Now you are screwed because you have no way to refer to that property name even when using the escape character ```\;``` or with quotes around the property name. The parser doesn't ignore the escaped or quoted semi-colon and thinks it is the end of the SQL statement.

example SQL (must use the OrientDB Studio, Browse tab)
```sql
CREATE PROPERTY V.test1 String
ALTER PROPERTY V.test1 NAME messedup; //note the semi-colon on the end
```
```sql
SELECT 'messedup;' FROM V  //error
SELECT 'messedup\;' FROM V  //error
SELECT messedup\; FROM V  //error
ALTER PROPERTY V.'messedup;' NAME newName; //error
DROP PROPERTY V.'messedup;' //error
```
Probably the same with CREATE CLASS but I don't dare try it because you won't be able to drop the class to recover from the bug.

I'm using OrientDB v2.1.9

**Comment by luigidellaquila** [comment]
Hi @Cyberider 

you should be able to drop it with a simple

```
drop property v.messedup;
```

Anyway, we are working to fix the problem

Thanks

Luigi

**Comment by Cyberider** [comment]
Hi @luigidellaquila
Thank you. I tried that and you are right, it does work. I didn't think to try the obvious :-) 

I didn't mention in my bug report a weird side effect because it was too much information.
Weird side effect:
if you have a console (CLI) connection to the db while you also have a Studio connection at the same time, and than you create a class property with the semi-colon (in Studio), the console caches the old property name and does not recognize the new property name with the semi-colon in it. I don't know why it would do any caching at all but that is the behaviour I experienced. The only way to bust the cache was for me to exit the OrientDB console session and start over again. Then than the console read the true property name with the semi-colon in it. I thought that was weird.

BTW what is the hidden field "originalName" used for? I thought maybe the console caching weirdness was using that field but "originalName" had the same value as "name". I've yet to see ```"originalName" ``` not have the same value as ```"name"```.
